src/lib/ngx-drop/file-drop.component.html

```diff
@@ -1,12 +1,14 @@
-<div [className]="customstyle" [class.over]="dragoverflag"
-    (drop)="dropFiles($event)"
-    (dragover)="onDragOver($event)" (dragleave)="onDragLeave($event)">
-    <div [className]="customContentStyle">
+<div [className]="dropZoneClassName"
+     [class.ngx-file-drop__drop-zone--over]="isDraggingOverDropZone"
+     (drop)="dropFiles($event)"
+     (dragover)="onDragOver($event)"
+     (dragleave)="onDragLeave($event)">
+    <div [className]="contentClassName">
         <ng-content></ng-content>
-        {{headertext}}
+        <div *ngIf="dropZoneLabel" class="ngx-file-drop__drop-zone-label">{{dropZoneLabel}}</div>
         <div *ngIf="showBrowseBtn">
-            <input type="file" #fileSelector [accept]="accept" (change)="uploadFiles($event)" multiple style="display: none;" />
```

Comment:
I think we need to keep the accept params, so the people could restrict the accepted files

Reply:
You are correct. This is a mistake on my part.